tests/testthat/test-tag_examplesShinylive.R

```diff
@@ -1,4 +1,5 @@
 test_that("examplesShinylive tag - errors - missing @examples", {
+  skip_if_not(packageVersion("roxygen2") >= "7.3.0")
```

Comment:
I was a bit confused at first about why the version specified here is different from the one in the DESCRIPTION file, but I think the reason we’re checking the `roxygen2` version is because a previous version produces different outputs, which affects the snapshot tests.

Should we consider not using snapshot tests at all and just rely on regex instead?

Reply:
yeah you got it right

the newer version of `roxygen2` returns the message whereas the older one returns the warning instead. All "positive" test cases are working identical.

I was thinking about regex but this will anyway ends-up with a condition on the roxygen package version - if (condition) then capture message else capture warning. Skipping was simpler.